mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -361,13 +362,27 @@ class DotConverter final : public OpConversionPattern<migraphx::DotOp> {
     auto mop = rewriter.create<tosa::MatMulOp>(loc, newOutType, in_A, in_B);
 
     // Convert optional attributes
-    if (auto attr = op->getAttrOfType<BoolAttr>("xdlopsV2"))
+    if (auto attr = (*op).template getAttrOfType<BoolAttr>("xdlopsV2"))
       mop->setAttr("xdlopsV2", attr);
-    if (auto attr = op->getAttrOfType<StringAttr>("perf_config"))
+    if (auto attr = (*op).template getAttrOfType<BoolAttr>("perf_config"))
```

Comment:
Not sure why these lines need to change. And this should still be a StringAttr right?

Reply:
Yeah, perf_config is a string, not a boolean

Reply:
I must be drunk when I write this.

Addressed in c31d296